frappe/model/document.py

```diff
@@ -1202,8 +1202,8 @@ def round_floats_in(self, doc, fieldnames=None):
 			doc.set(fieldname, flt(doc.get(fieldname), self.precision(fieldname, doc.parentfield)))
 
 	def get_url(self):
-		"""Returns Desk URL for this document. `/app/Form/{doctype}/{name}`"""
-		return "/app/Form/{doctype}/{name}".format(doctype=self.doctype, name=self.name)
+		"""Returns Desk URL for this document. `/app/{doctype}/{name}`"""
+		return "/app/{doctype}/{name}".format(doctype=self.doctype, name=self.name)
```

Comment:
LGTM. While you're at it, convert this to f-strings?

Reply:
I did the changes, Please review.